src/main.js

```diff
@@ -1,6 +1,8 @@
 import Vue from "vue";
 import App from "./App.vue";
 import "./filters";
+import "prismjs";
+import "prismjs/themes/prism-tomorrow.css";
```

Comment:
🤩 Looks really great. Tomorrow night is a great color theme!

Reply:
It is, right?!